5.0/en/0x02-Preface.md

```diff
@@ -1,57 +1,29 @@
 
-### Clear Set of Requirements
+* Refined Scope and Focus: The standard now aligns more directly with the foundational pillars in its name: Application, Security, Verification, and Standard. Requirements have been rewritten to emphasize the prevention of security flaws rather than mandating specific technical implementations. Each requirement is now crafted to be self-explanatory, highlighting its rationale.
 
-The requirements for version 5.0 were prepared based on the following considerations:
+* Support for Documented Security Decisions: ASVS 5.0 introduces requirements for documenting key security decisions. This enhances traceability and supports context-sensitive implementations, allowing organizations to tailor their security posture to their specific specific needs and risks.
 
-* Aggressively deduplicate requirements to avoid controls or concepts being split across multiple places.
-* Clarify unclear or non-actionable requirement text.
-* Add new requirements to cover areas of particular concern, such as permissions, tokens, and cryptography.
-* Introduce new chapters and sections for areas that might not apply to all applications yet remain security-sensitive (e.g., OAuth and WebSockets).
+* Updated Levels: While ASVS retains its three-tier model, the level definitions have evolved to make the ASVS easier to adopt. Level 1 now serves as a foundational entry point focused on primary defenses. Level 2 represents a comprehensive viewo of standard security practices, and Level 3 addresses advanced, high-assurance requirements.
 
-### Clarifying the Scope of the Standard
+* Restructured and Expanded Content: ASVS 5.0 now includes approximately 350 requirements across 17 chapters. Chapters have been reorganized for clarity and usability. A two-way mapping between v4.0 and v5.0 is provided to facilitate migration.
 
-It is important that all requirements are relevant to the defined scope of the standard and are worded consistently with its goals. The guidelines for this were:
+## Looking ahead
 
-* Ensure that all requirements are within the scope of a web application or service.
-* Verify that requirements align with the ASVS name, specifically:
-    * **Application** – Requirements are at the application level and are the responsibility of application developers.
-    * **Security** – Requirements are clearly necessary to secure the application.
-    * **Verification** – Requirements are written with a clear, verifiable goal.
-    * **Standard** – Requirements exhibit clear consistency and structure, as expected from a standard.
+The ASVS is not a static document. Although Version 5.0 is a major release, development continues. This release allows the wider community to benefit from the improvements and additions which have been accumulated but also lays the groundwork for future enhancements. This could include community-driven efforts to create implementation and verification guidance built on top of the core requirement set.
```

Comment:
"The ASVS is not a static document" - it may send a wrong signal like ASVS is a living standard. I would like to use something I used in my proposed version with the message, that "Security can not be never ready, so can not ASVS be never ready"

Reply:
How about:

"Just as securing an application is never truly finished, neither is the ASVS."